src/Mallos.Searchable/Mallos.Searchable.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Have to solve the issue where the Mallos.Searchable.CodeAnalyzer project can reference this by only `netstandard2.0`. 

I would like to add .NET 5 support here too, to be able to work with records.